training/src/app/modules/chuck-norris/components/joke-generator/joke-generator.component.html

```diff
@@ -0,0 +1,34 @@
+<div [ngClass]="{'search-style': isSearchMode}" *ngIf="isSearchMode" class="buttons-container">
+    <a>Key word(s)</a>
+    <app-search (generateJokesOnSearchMode)="onSearch($event)"
+                (generateJokesOnClickMode)="this.searchKey = $event"
+    ></app-search>
+    <button [ngClass]="{'disabled-style': this.searchKey.length < 3}"
```

Comment:
same logic here and in next line ( >3 ...).
how about creating a method "isButtonDisabled"? and move logic there